subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -39,6 +42,14 @@
 					   PDU_ISO_SEG_TIMEOFFSET_SIZE)
 #endif /* CONFIG_BT_CTLR_ADV_ISO || CONFIG_BT_CTLR_CONN_ISO */
 
+/* Defined the wrapping point and mid point in the range of time input values,
+ * which depend on range of the controller's clock in microseconds.
+ */
+#define ISOAL_TIME_WRAPPING_POINT         (HAL_TICKER_TICKS_TO_US(HAL_TICKER_CNTR_MASK))
+#define ISOAL_TIME_MID_POINT              (ISOAL_TIME_WRAPPING_POINT / 2)
+#define ISOAL_TIME_SPAN_FULL              (ISOAL_TIME_WRAPPING_POINT + 1)
+#define ISOAL_TIME_SPAN_HALF              (ISOAL_TIME_SPAN_FULL / 2)
```

Comment:
Why is `SPAN_HALF` not `ISOAL_TIME_SPAN_FULL / 2`?

Reply:
Sorry but I don't think that I understood the question. It is ISOAL_TIME_SPAN_FULL / 2.

Reply:
Yes, sorry, I misread :|